### PR TITLE
Improve instrument builder test plan

### PR DESCRIPTION
**Pull request by coreyAbout**
## Brief summary of changes
Group validate process together to improve the test work flow

#### Testing instructions (if applicable)

1. Read the TestPlan item 1 to item 8.

#### Links to related tickets (GitHub, Redmine, ...)
fixed:
#5494 
#5493 
#5488 
#5484 
*

### CODE REVIEW

**Comment by christinerogers** [comment]
Thanks for the pushes today @coreyAbout  
Please also update your issues with the label `PR Sent` (where applicable), and update the Testing-Project card?  Then I'll know changes here are done and I'll review.

**Comment by coreyAbout** [comment]
Doing it. Thanks @christinerogers for going through all the issues and provide valuable input.

File: modules/instrument_builder/test/TestPlan.md [hunk 1]
```diff
@@ -1,41 +1,35 @@
...
+    * 1.i.2 Validate adding range 
+  * 1.j Add Blank Line 
+  * 1.k Add Page break
+2.  Edit Question Name and Question Text in the table directly and check if it sticks.
+3.  Validate Instrument (Does not save if there is a same QuestionName twice)
+4.  Save Instrument file
+5.  Validate that not_answered option is added in the linst file for 1.e, 1.f, 1.g, 1.h
+6.  Preview Instrument [not available now].
+7.  Add Rules [not available now].
+8.  Save Rules file [not available now].
```
Comment (johnsaigle) [inline modules/instrument_builder/test/TestPlan.md]: Similar to the other PR, if these features don't exist yet then they shouldn't appear in the test plan. Instead a proposal should be created via Issues.

Comment (coreyAbout) [reply modules/instrument_builder/test/TestPlan.md]: Please ask @christinerogers for that. She requested to keep them.

**Comment by christinerogers** [comment]
Hi @coreyAbout -- a reminder, as we discussed in detail, Zaliqa is re-building the Instrument Builder for release asap, so these items would need to be restored to the test plan anyway in the near future.  
(This is why "not currently available" is not valid for the issue tracker #5540 - that feature was just plain retired.)  

cc @johnsaigle  If you/Dave prefer deleting them now, I'm totally fine with that.  A fresh test plan should accompany Zaliqa's commit anyway.

**Comment by johnsaigle** [comment]
Yeah I'm not even sure the new instrument builder would have the same interface And you're right, a new test plan would be needed.

I think these lines should be deleted.

**Comment by christinerogers** [comment]
@coreyAbout @johnsaigle I did 2 quick commits here for this test plan:
 - [c7fa0e](https://github.com/aces/Loris/pull/5534/commits/c7fa0e) deleted the "not functioning now" feature points

- [339df8](https://github.com/aces/Loris/pull/5534/commits/c7fa0e) separated the point about adding and testing Rules, since prior mentions are removed in Corey's commit(s). 

Btw, @coreyAbout  did you test any Rules for your linst file when you installed it? 

@johnsaigle i'm requesting your review since I'm now compromised :)

**Comment by driusan** [comment]
It looks like John already approved this?

Can someone give me a description of what the most up to date version of this changes so I can squash it into a commit message?

**Comment by johnsaigle** [comment]
@driusan 'Remove references to nonextant feature; improve readability'

And yes it's good to go

**Pull request merged**